railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -21,6 +21,11 @@ source 'https://rubygems.org'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+<%- if options.api? -%>
+# Use ActiveModelSerializers to serialize JSON responses
+gem 'active_model_serializers', '~> 0.10.0.rc1'
```

Comment:
@spastorino posted to https://groups.google.com/d/topic/rubyonrails-core/K8t4-DZ_DkQ/discussion